gnutls build broken

https://oss-fuzz-build-logs.storage.googleapis.com/index.html

```
Step #4: make[4]: Leaving directory '/src/gmp'
Step #4: make[3]: Leaving directory '/src/gmp'
Step #4: make[2]: Leaving directory '/src/gmp'
Step #4: make[1]: Leaving directory '/src/gmp'
Step #4: + cd /src/libtasn1
Step #4: + CFGFLAGS='--disable-gcc-warnings --disable-gtk-doc --disable-gtk-doc-pdf --disable-doc   --disable-shared --enable-static --prefix=/src/deps'
Step #4: + make bootstrap
Step #4: make: *** No rule to make target 'bootstrap'.  Stop.
Finished Step #4
ERROR
ERROR: build step 4 "gcr.io/oss-fuzz/gnutls" failed: exit status 2
```

@nmav - can you please take a look.

Ok let me try

```
Step #4: make[4]: Leaving directory '/src/gmp'
Step #4: make[3]: Leaving directory '/src/gmp'
Step #4: make[2]: Leaving directory '/src/gmp'
Step #4: make[1]: Leaving directory '/src/gmp'
Step #4: + cd /src/libtasn1
Step #4: + CFGFLAGS='--disable-gcc-warnings --disable-gtk-doc --disable-gtk-doc-pdf --disable-doc   --disable-shared --enable-static --prefix=/src/deps'
Step #4: + make bootstrap
Step #4: make: *** No rule to make target 'bootstrap'.  Stop.
Finished Step #4
ERROR
ERROR: build step 4 "gcr.io/oss-fuzz/gnutls" failed: exit status 2
```

--- a/projects/gnutls/Dockerfile
+++ b/projects/gnutls/Dockerfile
@@ -39,7 +39,7 @@ RUN git clone git://git.savannah.gnu.org/gnulib.git
 RUN git clone --depth=1 https://git.savannah.gnu.org/git/libunistring.git
 RUN git clone --depth=1 https://gitlab.com/libidn/libidn2.git
 RUN hg clone https://gmplib.org/repo/gmp/ gmp
-RUN git clone --depth=1 https://git.savannah.gnu.org/git/libtasn1.git
+RUN git clone --depth=1 https://gitlab.com/gnutls/libtasn1.git
 RUN git clone --depth=1 https://git.lysator.liu.se/nettle/nettle.git
 
 RUN git clone --depth=1 --recursive https://gitlab.com/gnutls/gnutls.git

